### PR TITLE
#1455 Display recipient info popup when user tap recipients

### LINKED ISSUE #1455: add a way to see recipient email address on compose
Ever since we have used names for recipients, it's no longer possible to figure out which email address I'm emailing to on compose screen.

Gmail app has a solution for this - when I tap the recipient bubble, it shows me the name and email address.

I'd like something similar.

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR changed way how recipients are displayed when user taps recipient

close #1455 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by ioanmo226** [comment]
@tomholub MVP version is ready (There are still many bugs and no functions/tests are added yet).
I just want you to confirm if I'm in the correct direction.

**Comment by tomholub** [comment]
Yes - that was exactly what I meant. Looks good.

**Comment by tomholub** [comment]
When you do copy-all, I suppose it could be comma-separated list of recipients.

**Comment by ioanmo226** [comment]
Yeah, correct. Okay got it.

> When you do copy-all, I suppose it could be comma-separated list of recipients. But you don't necessarily have to implement copy-all. Just copy would do.


By the way, I think there should also be button which will refresh recipient key info. (something like `Refresh Key`).
What do you think?

**Comment by ioanmo226** [comment]
@tomholub Please confirm above and let me know what other features you want me to add in that popup.

**Comment by tomholub** [comment]
> @tomholub Please confirm above and let me know what other features you want me to add in that popup.

Less is better. We can always add more later if we deem it needed.

**Comment by ioanmo226** [comment]
Okay, got it

**Comment by tomholub** [comment]
I just thought of (needed) another item on that menu - a way to edit the recipient. For example, I made a type in the recipient, but I don't want to remove it and re-type the whole thing. I'll file another issue.

File: appium/tests/screenobjects/new-message.screen.ts [hunk 1]
```diff
@@ -236,7 +246,17 @@ class NewMessageScreen extends BaseScreen {
     await this.showRecipientInputIfNeeded();
     const addedRecipientEl = await $(`~aid-${type}-${order}-label`);
     await ElementHelper.waitAndClick(addedRecipientEl);
-    await driver.sendKeys(['\b']); // backspace
+    await ElementHelper.waitAndClick(await this.recipientPopupRemoveButton);
+    await ElementHelper.waitElementInvisible(addedRecipientEl);
```
Comment (tomholub) [inline appium/tests/screenobjects/new-message.screen.ts:250]: Both backspace and tap+remove should work, so we should ideally test both. Let's also add back the test for `await driver.sendKeys(['\b']); // backspace` here, in another issue.

File: appium/tests/specs/live/composeEmail/CheckRecipientPopup.spec.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    await NewMessageScreen.setAddRecipient(recipient1);
+    await NewMessageScreen.setAddRecipient(recipient2);
+    await NewMessageScreen.setAddRecipient(recipient3);
+
+    await NewMessageScreen.checkCopyForAddedRecipient(recipient1, 0);
+
+    await NewMessageScreen.deleteAddedRecipient(2);
+    await NewMessageScreen.deleteAddedRecipient(1);
+    await NewMessageScreen.deleteAddedRecipient(0);
```
Comment (tomholub) [inline appium/tests/specs/live/composeEmail/CheckRecipientPopup.spec.ts:27]: In all of the popups, we should also check it displays expected email and name. Can be in another issue.

**Pull request merged**